crates/store/re_chunk_store/src/store.rs

```diff
@@ -659,8 +666,9 @@ impl ChunkStore {
         let is_indicator = component_name.is_indicator_component();
 
         use re_types_core::Archetype as _;
-        let is_tombstone =
-            re_types_core::archetypes::Clear::all_components().contains(component_name);
+        let is_tombstone = re_types_core::archetypes::Clear::all_components()
+            .iter()
+            .any(|descr| descr.component_name == *component_name);
```

Comment:
This is one of many many _many_ examples where we still have component-name based logic that needs to be migrated to component-descriptor based logic.

The goal of this PR is to get tags in and out, not migrate everything that exists in one go (that PR is big enough as is!).